modules/core/src/main/java/org/apache/ignite/internal/processors/dr/IgniteDrDataStreamerCacheUpdater.java

```diff
@@ -69,6 +72,8 @@ public class IgniteDrDataStreamerCacheUpdater implements StreamReceiver<KeyCache
 
             CacheObjectContext cacheObjCtx = cache.context().cacheObjectContext();
 
+            IgniteLogger log = cache0.unwrap(Ignite.class).log();
+
```

Comment:
Let's cache logger in a field.
Like this
```
@LoggerResource
IgniteLogger log;
```
or, otherwise, maybe making field `transient` could be helpful.